Storage/Storage/Model/ShippingLabel+CoreDataClass.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
```

Comment:
LOC-conservation trick for the future: We don't actually need to import `Foundation` if there are other imports that import it already, like `CoreData`. 

There! I saved you one line of change! 🤣

This is hinted in: https://github.com/raywenderlich/swift-style-guide/#minimal-imports

Reply:
haha, these files were automatically generated by Xcode I think I'll leave them as they are for now 🙂 guess Apple doesn't follow this style guide 🙄